backend/wmg/requirements.txt

```diff
@@ -0,0 +1,105 @@
+anndata==0.7.8
```

Comment:
Are these the reqs for the batch process? are any of them also needed by the api?

Reply:
is this the output of `pip --freeze`? Don't we just want the direct requirements in here?

Reply:
I couldn't figure out a working set of dependencies so I went to Madison's EC2 machine and did a `pip freeze`. I agree it would be a better idea to skim them to a minimal set of requirements, but that will take me a while. I might do that in a separate PR.